Dockerfile.openpilot_dev

```diff
@@ -0,0 +1,5 @@
+FROM openpilot_base:latest
```

Comment:
There's prebuilt image available at `ghcr.io/commaai/openpilot-base:latest`. It's x86 only, but we should be able to build and ship aarch64 ones on CI

Reply:
Yes, the issue is that this one doesn't take into account local changes, but that might be fine

Reply:
and yes, if we can build an aarch64 version in CI too that would be nice. Don't think there's github action runners for this yet, although we could use qemu?

Reply:
> Yes, the issue is that this one doesn't take into account local changes, but that might be fine

That doesn't happen that often, we rarely change setup scripts or add packages. IMO it's fine, we can always provide additional instructions for people which will want to use custom builds.

> Don't think there's github action runners for this yet, although we could use qemu?

Unfortunately they don't have them yet. QEMU is viable solution, just have to examine how slow it's gonna be.

Reply:
Ok, let's just support x86 for now then, and we can fix this later when GH finally releases ARM builders.